entrypoint.sh

```diff
@@ -5,5 +5,5 @@ set -e # fail on error
 # include hidden files 
 # https://askubuntu.com/questions/740805/how-can-i-remove-all-files-from-current-directory-using-terminal
 shopt -s dotglob
```

Comment:
I think the shopt call can also be removed, including with the comment about hidden files

Reply:
Agree.
Removed the unneeded shopt.